src/views/preview/preview.jsx

```diff
@@ -63,6 +68,14 @@ class Preview extends React.Component {
         };
         this.getExtensions(this.state.projectId);
         this.addEventListeners();
+        /* In the beginning, if user is on mobile and landscape, go to fullscreen */
+        if (this.props.playerMode && isMobileDevice()) {
+            if (screen.orientation.type === 'landscape-primary') {
```

Comment:
This way of detecting landscape will need additional testing! Esp. on iPad + Safari